css/app.css

```diff
@@ -14,6 +14,46 @@ select:focus {
 /************** */
 /** My Styles */
 /************** */
+
+body {
```

Comment:
Stying the `body` would be visible before `login` as well. So, if you want your changes to be visible after `logged in`, it is recommended to work with `signedIn` div instead

Reply:
ok, great hint, I will update it.